scripts/build_utils.sh

```diff
@@ -128,3 +128,78 @@ check_binary_existence () {
     fi
 
 }
+
+
+submit_build_status() {
+    # A helper script to post (create) the status of a build in shaman
+    # 'state' can be either 'failed' or 'started'
+    # 'project' is used to post to the right url in shaman
+    http_method=$1
+    state=$2
+    project=$3
+    distro=$4
+    distro_version=$5
+    distro_arch=$6
+    cat > $WORKSPACE/build_status.json << EOF
+{
+    "extra":{
+        "version":"$vers",
+        "root_build_cause":"$ROOT_BUILD_CAUSE",
+        "node_name":"$NODE_NAME",
+        "job_name":"$JOB_NAME",
+        "url":"$BUILD_URL",
+        "build_user":"$BUILD_USER",
+        "log_url":"$BUILD_URL/consoleFull"
+    },
+    "state":"$state",
+    "distro":"$distro",
+    "distro_version":"$distro_version",
+    "distro_arch":"$distro_arch",
+    "branch":"$BRANCH",
```

Comment:
Is this the parsed version of branch that removes any extra forward slashes in the branch name? I think we'd want to get this the same as what gets used in chacra.